pkg/scheduler/api/devices/ascend/ascend310p/vnpu/type.go

```diff
@@ -0,0 +1,641 @@
+	UpdateOperator = "update"
+
+	// CmConsumer who uses these configmap
+	CmConsumer = "mx-consumer-volcano"
+	// NormalCmConsumer normal who uses these configmap
+	NormalCmConsumer = "mx-consumer-cim"
+	// CmConsumerValue the value only for true
+	CmConsumerValue = "true"
+	// ClusterDeviceInfo the name of cluster device info configmap
+	ClusterDeviceInfo = "cluster-info-device-"
+	// ClusterNodeInfo the name of cluster node info configmap
+	ClusterNodeInfo = "cluster-info-node-cm"
+	// ClusterSwitchInfo the name of cluster switch info configmap
+	ClusterSwitchInfo = "cluster-info-switch-"
+
+	// Pod910DeviceKey pod annotation key, for generate 910 hccl rank table
+	Pod910DeviceKey = "ascend.kubectl.kubernetes.io/ascend-910-configuration"
+	// PodPredicateTime set pod PodPredicateTime for using by device-plugin.
+	PodPredicateTime = "predicate-time"
+	// NodeNotMeetTopologyWarning node not satisfy the schedulable topology warning.
+	NodeNotMeetTopologyWarning = "the npus on this node don't satisfy the schedulable topology"
+	// ArgumentError argument nil error.
+	ArgumentError = "invalid argument"
+	// JobKindKey for define the Job kind:ascend-310P, ascend-910
+	JobKindKey = "ring-controller.atlas"
+	// JobKind910Value in ring-controller.atlas.
+	JobKind910Value = "ascend-910"
+	// JobKind310Value in ring-controller.atlas.
+	JobKind310Value = "ascend-310"
+	// JobKind310PValue 310p ring controller name
+	JobKind310PValue = "ascend-310P"
+	// JobKind910BValue 910B ring controller name
+	JobKind910BValue = "ascend-910b"
+	// DistributedJobKey flag for distributed job
+	DistributedJobKey = "distributed-job"
+	// DistributedJobValue indicate distributed job
+	DistributedJobValue = "true"
+	// StandaloneJobValue indicate standalone job
+	StandaloneJobValue = "false"
+
+	// SuperPodAnnoKey annotation key of super pod
+	SuperPodAnnoKey = "sp-block"
+	// DistributedInferKey distributed infer
+	DistributedInferKey = "distributed"
+	// DistributedInferLabel true or false
+	DistributedInferLabel = "true"
+	// OperatorNameLabelKey pod label key for acjob operator name
+	OperatorNameLabelKey = "training.kubeflow.org/operator-name"
+)
+
+const (
+	// AffScore0 value 0 for scored.
+	AffScore0 = iota
+	// AffScore1 value 1 for scored.
+	AffScore1
+	// AffScore2 value 2 for scored.
+	AffScore2
+	// AffScore3 value 3 for scored.
+	AffScore3
+	// AffScore4 value 4 for scored.
+	AffScore4
+	// AffScore5 value 5 for scored.
+	AffScore5
+	// AffScore6 value 6 for scored.
+	AffScore6
+	// AffScore7 value 7 for scored.
+	AffScore7
+	// AffScore8 value 8 for scored.
+	AffScore8
+	// AffScore15 value 15 for scored.
+	AffScore15
+)
+
+const (
+	// JobNotEnqueue job enqueue failed
+	JobNotEnqueue = -1
+	// JobEnqueue job enqueue success
+	JobEnqueue = 1
+	// JobEnqueueSkip skip the judgement of ascend-volcano-plugin in the job enqueue phase
+	JobEnqueueSkip = 0
+	// PodGroupInqueue the pg Inqueue status
+	PodGroupInqueue = "Inqueue"
```

Comment:
The `volcano/apis` dependency already contains these phase, should not define duplicate consts